source/administration/telemetry.rst

```diff
@@ -40,15 +40,15 @@ Server Configuration Settings
+    **ServiceSettings**: enum WebserverMode, bool EnableSecurityFixAlert, bool EnableInsecureOutgoingConnections, bool EnableIncomingWebhooks, bool EnableOutgoingWebhooks, bool EnableCommands, bool EnableOnlyAdminIntegrations, bool EnablePostUsernameOverride, bool EnablePostIconOverride, bool EnableCustomEmoji, enum RestrictCustomEmojiCreation, bool EnableTesting, bool EnableDeveloper, bool EnableMultifactorAuthentication, bool EnableOAuthServiceProvider, enum ConnectionSecurity, bool UseLetsEncrypt, bool Forward80To443, enum ConnectionSecurity, bool EnforceMultifactorAuthentication, enum RestrictPostDelete, bool AllowEditPost, bool EnableUserTypingMessages, bool EnablePostSearch, bool EnableUserStatuses, bool EnableChannelViewMessages, bool EnableEmojiPicker, bool TeammateNameDisplay; **TeamSettings**: bool EnableUserCreation, bool EnableTeamCreation, bool RestrictTeamNames, enum RestrictTeamInvite, enum RestrictPublicChannelManagement, enum RestrictPrivateChannelManagement, enum RestrictPublicChannelCreation, enum RestrictPrivateChannelCreation, enum RestrictPublicChannelDeletion, enum RestrictPrivateChannelDeletion, enum RestrictPrivateChannelManageMembers, bool EnableOpenServer, bool EnableCustomBrand, bool RestrictDirectMessage; **SqlSettings**: enum DriverName, bool Trace; **LogSettings**: bool EnableConsole, enum ConsoleLevel, bool EnableFile, enum FileLevel, bool EnableWebhookDebugging; **PasswordSettings**: bool Lowercase, bool Number, bool Uppercase, bool Symbol; **FileSettings**: bool EnablePublicLink, enum DriverName, bool AmazonS3SSL, bool EnableFileAttachments, bool AmazonS3SignV2; **EmailSettings**: bool EnableSignUpWithEmail, bool EnableSignInWithEmail, bool EnableSignInWithUsername, bool RequireEmailVerification, bool SendEmailNotifications, enum ConnectionSecurity, bool SendPushNotifications, enum PushNotificationContents, bool EnableEmailBatching, bool SkipServerCertificateVerification; **RateLimitSettings**: bool EnableRateLimiter, bool VaryByRemoteAddr; **PrivacySettings**: bool ShowEmailAddress, bool ShowFullName; **GitLabSettings**: bool Enable; **GoogleSettings**: bool Enable; **Office365Settings**: bool Enable; **LdapSettings**: bool Enable, enum ConnectionSecurity, bool SkipCertificateVerification; **ComplianceSettings**: bool Enable, bool EnableDaily; **LocalizationSettings**: enum DefaultServerLocale, enum DefaultClientLocale, enum AvailableLocales; **SamlSettings**: bool Enable, bool Verify, bool Encrypt; **ClusterSettings**: bool Enable, bool UseIpAddress, bool UseExperimentalGossip, bool ReadOnlyConfig; **MetricsSettings**: bool Enable; **WebrtcSettings**: bool Enable, **AnnouncementSettings**: bool EnableBanner, bool AllowBannerDismissal
```

Comment:
Can you double-check the SQL settings? This is what's in `diagnostics.go`:
``` go
SendDiagnostic(TRACK_CONFIG_SQL, map[string]interface{}{
		"driver_name":                 utils.Cfg.SqlSettings.DriverName,
		"trace":                       utils.Cfg.SqlSettings.Trace,
		"max_idle_conns":              utils.Cfg.SqlSettings.MaxIdleConns,
		"max_open_conns":              utils.Cfg.SqlSettings.MaxOpenConns,
		"data_source_replicas":        len(utils.Cfg.SqlSettings.DataSourceReplicas),
		"data_source_search_replicas": len(utils.Cfg.SqlSettings.DataSourceSearchReplicas),
		"query_timeout":               *utils.Cfg.SqlSettings.QueryTimeout,
	})
```

but this is what the **SqlSettings** are:

`**SqlSettings**: enum DriverName, bool Trace;`